docs/application/web/api/5.5/device_api/mobile/tizen/notification.html

```diff
@@ -2344,7 +2344,6 @@ <h2 id="full-webidl">4. Full WebIDL</h2>
 <pre class="webidl prettyprint">module Notification {
   typedef DOMString NotificationId;
   enum NotificationType { "STATUS" };
-  enum StatusNotificationType { "SIMPLE", "THUMBNAIL", "ONGOING", "PROGRESS" };
```

Comment:
Actually the point of my question was the ground of this removal, as I couldn't see the reason of this removal from the PR description. However, if so, I'll accept.

Another Question. From when this enum should be removed? I can see this enum in the same file of master branch, which means in tizen 5.0 we have same bug with it.

Reply:
I decided that this started to get a bit overcomplicated and removed changes unrelated to the server icon URI. We will create a separate ACR to merge those changes later on.